File: MPI/mpicheck/tests/functions/2.0/MPI_File_write_ordered_end.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_File var_0;
    const void *var_1;
    MPI_Status *var_2;
    int ret;
    /* calls */
    ret = MPI_File_write_ordered_end(var_0, var_1, var_2);
    ret = PMPI_File_write_ordered_end(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_File_write_ordered_end.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       TYPE(INTEGER) var_1(10)
       INTEGER var_2(10)
       INTEGER var_3
        call mpi_file_write_ordered_end(var_0, var_1, var_2, var_3)
       call pmpi_file_write_ordered_end(var_0, var_1, var_2, var_3)
        end program main
    